### PR TITLE
Fixes #121 - Add a test to open a website from the details page

### LINKED ISSUE #121: Add a test to open a website from the details page
## Steps to reproduce:
1. Open marketplace-dev app on your firefoxOS device
2. Go to New tab
3. Filter the content to show only Websites
4. Click on one of the listed websites to load the details page
5.  Click "Open"

## Expected behavior:
The website loads in a new tab on the browser

### PR DESCRIPTION
**Pull request by bobsilverberg**
@davehunt r?

### CODE REVIEW

File: marketplacetests/marketplace/pages/new.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    _app_locator = (By.CSS_SELECTOR, 'ul.app-list li')
+
+    @property
+    def app_list(self):
+        self.wait_for_element_displayed(*self._app_locator)
+        apps = self.marionette.find_elements(*self._app_locator)
+        return [Result(self.marionette, app) for app in apps]
+
+    def select_content(self, type):
```
Comment (davehunt) [inline marketplacetests/marketplace/pages/new.py]: Let's avoid using `type` as that's reserved.

File: marketplacetests/marketplace/pages/app_details.py [hunk 2]
```diff
@@ -34,6 +35,13 @@ def first_review_body(self):
     def first_review_rating(self):
         return int(self.marionette.find_element(*self._first_review_locator).get_attribute('data-rating'))
 
+    @property
+    def open_website_href(self):
```
Comment (davehunt) [inline marketplacetests/marketplace/pages/app_details.py]: This sounds like it would open the website, maybe drop `open_` from the method name?

**Comment by davehunt** [comment]
r+ with comments addressed

**Comment by bobsilverberg** [comment]
Thanks @davehunt.

**Pull request merged**